lms/djangoapps/teams/static/teams/js/views/team_profile_header_actions.js

```diff
@@ -9,7 +9,7 @@
         function (Backbone, _, gettext, TeamUtils, teamProfileHeaderActionsTemplate) {
             return Backbone.View.extend({
 
-                errorMessage: gettext("An error occurred. Try again."),
+                errorMessage: _.escape(gettext("An error occurred. Try again.")),
```

Comment:
http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#underscore-template-files
Although here it only says for underscore templates but i thought it might true for js files. 
@cahrens @sarina